aries_vcx_core/src/ledger/indy_vdr_ledger.rs

```diff
@@ -373,24 +378,27 @@ where
         submitter_did: &str,
         endorser_did: Option<String>,
     ) -> VcxCoreResult<()> {
-        let request = self._build_schema_request(submitter_did, schema_json)?;
-        let request = _append_txn_author_agreement_to_request(request).await?;
-        self._sign_and_submit_request(submitter_did, request).await?;
-        Ok(())
+        let mut request = self._build_schema_request(submitter_did, schema_json)?;
+        request = _append_txn_author_agreement_to_request(request).await?;
+        if let Some(endorser_did) = endorser_did {
```

Comment:
this endorsing sequence is missing for other `publish_*` methods, but I see that we don't even pass endorses_did Option in those. Okay, I guess that could be another github issue (perhaps even good-first-issue, should be fairly simple)